src/java.management/share/classes/java/lang/management/ThreadMXBean.java

```diff
@@ -31,12 +31,11 @@
  * The management interface for the thread system of the Java virtual machine.
  *
  * <p> {@code ThreadMXBean} supports monitoring and management of
- * <a href="{@docRoot}/java.base/java/lang/Thread.html#platform-threads">
- * platform threads</a> in the Java virtual machine. Platform threads are
+ * {@linkplain Thread##platform-threads
+ * platform threads} in the Java virtual machine. Platform threads are
  * typically mapped to kernel threads scheduled by the operating system.
  * {@code ThreadMXBean} does not support monitoring or management of
- * <a href="{@docRoot}/java.base/java/lang/Thread.html#virtual-threads">
- * virtual threads</a>.
+ * {@linkplain Thread##virtual-threads virtual threads}.
```

Comment:
The use of ## looks fine but would you mind re-flowing the javadoc in this paragraph as the line break after platform-threads looks accidental. This paragraph is mostly new text since Java 19.